diffs/AaveV3Ethereum_PyUSDParametersOptimization_20250919_before_AaveV3Ethereum_PyUSDParametersOptimization_20250919_after.md

```diff
@@ -0,0 +1,96 @@
+## Reserve changes
+
+### Reserves altered
+
+#### PYUSD ([0x6c3ea9036406852006290770BEdFcAbA0e23A0e8](https://etherscan.io/address/0x6c3ea9036406852006290770BEdFcAbA0e23A0e8))
```

Comment:
I think test needs to be run again, as LTV 0 not showing on diff

Reply:
Error comes from this: https://github.com/bgd-labs/aave-proposals-v3/pull/839#discussion_r2393890056